crates/subspace-runtime/src/lib.rs

```diff
@@ -966,6 +970,9 @@ construct_runtime!(
         Democracy: pallet_democracy = 83,
         Preimage: pallet_preimage = 84,
 
+        // history seeding
```

Comment:
No need to add such comments, they are not required and do not help understanding the code